rust/xaynet-mobile/src/ffi/participant.rs

```diff
@@ -345,3 +355,93 @@ pub unsafe extern "C" fn xaynet_ffi_participant_set_model(
         ERR_SETMODEL_MODEL
     }
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn xaynet_ffi_participant_global_model(
+    participant: *mut Participant,
+    buffer: *mut c_void,
+    // len: c_uint,
```

Comment:
Not sure if we should keep the `len` and the `data_type` here.
both can be accessed through `participant.model_config().model_length` and `participant.model_config().mask_config.vect.data_type`.